src/components/randomDrinkCards/RandomDrinkCards.css

```diff
@@ -1,6 +1,24 @@
 .drink-card {
+   color: black;
+   background-color: azure;
    display:flex;
    flex-direction: column;
    align-items: center;
-   justify-content: center;
+   justify-content: baseline;
+   font-family: 'Montserrat', sans-serif;
+}
+
+.drink-image {
+   display: flex;
+   flex-direction: column;
+   height: 40%;
+   width: 40%;
+   margin-top: 10%;
+}
+
+.drink-card:hover {
+   cursor: pointer;
+   transform: scale(1.1);
+   transition: all .8s;
+   box-shadow: 15px 15px 10px rgba(0, 0, 0, 0.3);
```

Comment:
A lot of these styles I have not really played with before so I am interested to see what it looks like